client/src/app/pages/assessment/components/assessment-wizard/assessment-wizard.tsx

```diff
@@ -583,13 +583,13 @@ export const AssessmentWizard: React.FC<AssessmentWizardProps> = ({
             <ConfirmDialog
               title={t("dialog.title.leavePage")}
```

Comment:
title should probably be changed too. "Close Assessment"? or if you want to use existing message... maybe `dialog.title.discard` with Assessment for the `what`?

Reply:
Other possible confirm modal titles:

  - "Leave assessment"
  - "Confirm cancel"
  - "Close wizard"